src/Model.php

```diff
@@ -965,6 +965,13 @@ public function _unset(string $name)
      */
     public function addCondition($field, $operator = null, $value = null)
     {
+        // legacy OR support before Scope was introduced
+        if (func_num_args() === 1 && is_array($field) && count($field) === 1 && is_array(reset($field))) {
```

Comment:
we should probably drop the support later, now fully BC

Reply:
👍